Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY . $GOPATH/src/github.com/jimmidyson/configmap-reload
 
 RUN yum install -y golang make git && \
    cd $GOPATH/src/github.com/jimmidyson/configmap-reload && \
-   PATH=$PATH:$GOPATH/bin make out/configmap-reload-linux-amd64 GOPATH=$GOPATH && cp $GOPATH/src/github.com/jimmidyson/configmap-reload/out/configmap-reload-linux-amd64 /usr/bin/configmap-reload && \
+   PATH=$PATH:$GOPATH/bin make out/configmap-reload-linux-`go env GOARCH` GOPATH=$GOPATH && cp $GOPATH/src/github.com/jimmidyson/configmap-reload/out/configmap-reload-linux-`go env GOARCH` /usr/bin/configmap-reload && \
```

Comment:
Never ever use back-ticks, always use `$(go env GOATCH)` instead.

Reply:
Thanks @soltysh I'll fix and submit another PR. :)